website/docs/r/supervisor_namespace.html.markdown

```diff
@@ -0,0 +1,152 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_supervisor_namespace"
+sidebar_current: "docs-vcfa-resource-supervisor-namespace"
+description: |-
+ Provides a VMware Cloud Foundation Automation Supervisor Namespace. This can be used to create, modify, and delete Roles.
```

Comment:
```suggestion
 Provides a VMware Cloud Foundation Automation Supervisor Namespace. This can be used to create, modify, and delete Supervisor namespaces.
```

Reply:
Fixed.